addons/flink-kubernetes-operator/definitions/flink-cluster.cue

```diff
@@ -54,19 +54,30 @@ template: {
                 }
         }
         parameter: {
-                                nots: *"2"|string
-                                flinkVersion: *"v1_14"|string
-                                image: *"flink:latest"|string
+             // +usage=Specify the taskmanager.numberOfTaskSlots, e.g "2"
+                                nots: string
+             // +usage=Specify the flink cluster version, e.g "v1_14"
+                                flinkVersion: string
+             // +usage=Specify the image for flink cluster to run, e.g "flink:latest"
+                                image: string
+             // +usage=Specify the flink cluster name
                                 name: string
+             // +usage=Specify the namespace for flink cluster to install
                                 namespace: string
-
-                                jarURI: *"local:///opt/flink/examples/streaming/StateMachineExample.jar"|string
-                                parallelism: *2|int
-                                upgradeMode: *"stateless"| string
-                                replicas: *1|int
-                                jmcpu: *1|int
```

Comment:
why remove default values?

Reply:
Users need to know the related parameters for better using.